src/utils/Chart.js

```diff
@@ -0,0 +1,35 @@
+const d3 = require('d3');
+
+const Chart = {
+  getDataMinMaxValues (data, axis) {
+    const max = d3.max(data, d => {
+      return Math.ceil(d[axis] / 1000) * 1000;
+    });
+
+    const min = d3.min(data, d => {
+      return Math.floor(d[axis] / 1000) * 1000;
+    });
+
+    return { min, max };
+  },
+
+  getAxisTickValues (data, axis) {
+    const minMaxValues = this.getDataMinMaxValues(data, axis);
+    const range = minMaxValues.max - minMaxValues.min;
+    const tempStep = range / 6;
```

Comment:
6 is the number of ticks that we would like displayed.  I'll move this to a variable so it is discoverable.